### PR TITLE
Translate/danhpaiva

### PR DESCRIPTION
**Pull request by danhpaiva**
Translation correction of some Portuguese words

### CODE REVIEW

File: 1-getting-started-lessons/2-github-basics/translations/README.pt.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Digamos que você tenha uma pasta localmente com algum projeto de código e dese
         modified:   file2.txt
    ```
 
-   Geralmente o comando `git status` diz a você coisas como quais arquivos estão prontos para serem _salvos_ para o repo ou tem alterações que você pode querer persistir.
+   Geralmente o comando `git status` diz a você coisas como: quais arquivos estão prontos para serem _salvos_ para o repo ou tem alterações que você pode querer persistir.
```
Comment (jlooper) [inline 1-getting-started-lessons/2-github-basics/translations/README.pt.md:83]: Is this colon needed here?

**Review by jlooper — COMMENTED** [review]
just one question on the colon

File: 1-getting-started-lessons/2-github-basics/translations/README.pt.md (hunk 1 above)
Comment (danhpaiva) [reply 1-getting-started-lessons/2-github-basics/translations/README.pt.md:83]: Yes, it exemplifies what git can accomplish.

Comment (danhpaiva) [reply 1-getting-started-lessons/2-github-basics/translations/README.pt.md:83]: @jlooper Would you like to remove the colon?

Comment (danhpaiva) [reply 1-getting-started-lessons/2-github-basics/translations/README.pt.md]: Done!

**Review by jlooper — APPROVED** [review]

**Pull request merged**